cpp/BUILDING.md

```diff
@@ -37,9 +37,7 @@ unnecessary.
 Ice was extensively tested using the operating systems and compiler versions
 listed on [supported platforms][2].
 
-On Windows, the build requires a recent version of Visual Studio. When building
-with Visual Studio 2017 you have to install Desktop development with C++ workload
-and select the optional component Windows 8.1 SDK and UCRT SDK.
+On Windows, the build requires a recent version of Visual Studio 2022.
```

Comment:
Should be either "requires Visual Studio 2022" or "requires Visual Studio 2022 version XY or greater".